kernel_gateway/gatewayapp.py

```diff
@@ -174,6 +174,16 @@ def allow_origin_default(self):
     def expose_headers_default(self):
         return os.getenv(self.expose_headers_env, '')
 
+    x_headers_env = 'KG_X_HEADERS'
+    x_headers = Unicode(config=True,
+        help='Uses x-* header values for overriding the remote-ip, useful when application is behing a proxy. (KG_X_HEADERS env var)'
+    )
+
+    @default('x_headers')
+    def x_headers_default(self):
+        return os.getenv(self.x_headers_env, 'false')
```

Comment:
It may be a good idea to handle various formats for the env value, so something like this may work more reliably...
```
return bool(os.getenv(self.x_headers_env, 'false').lower() == 'true')
```